Mathlib/Order/Hom/Basic.lean

```diff
@@ -526,6 +526,11 @@ protected def withTopMap (f : α →o β) : WithTop α →o WithTop β :=
 
 end OrderHom
 
+-- See note [lower instance priority]
+instance (priority := 90) OrderHomClass.toOrderHomClassOrderDual [LE α] [LE β]
```

Comment:
I can't comment on these changes

Reply:
Do you mean about the lower instance priority?